requirements.in

```diff
@@ -1,7 +1,9 @@
--e git://github.com/mbertheau/python-emarsys@master#egg=python-emarsys
+-e git://github.com/mbertheau/python-emarsys@0.3#egg=python-emarsys
+
 # lowest supported django version
-Django<1.8
+Django>1.8
```

Comment:
Interesting. I was pretty sure that I tested it with the 1.8 line of Django and it didn't work either. I've explicitly pinned the 1.8.14 version and upwards together with 1.9.8 and upwards until 1.10 line for now.

The test suite will be run by tox for the Django 1.8 line and 1.9 line under Python 2.7 and 3.5 now. It was necessary to split the requirements to achieve this though. Pip doesn't like it that much when a dependency is specified twice it seems.